internal/commands/untrust_builder.go

```diff
@@ -0,0 +1,56 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+func UntrustBuilder(logger logging.Logger, cfg config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "untrust-builder <builder-name>",
+		Short: "Stop trusting builder",
+		Long:  "Stop trusting builder.\n\nWhen building with this builder, all lifecycle phases will be no longer be run in a single container using the builder image.",
+		Args:  cobra.MaximumNArgs(1),
```

Comment:
I believe there is an option for "exact" number of Args.

Reply:
I expect that's true, this implementation is consistent with `trust-builder` so we should update that too.

Reply:
The small side effect to using `cobra.ExactArgs(1)` is that the command will exit with non-zero status, which is probably fine, but does lead to a small change in functionality.

Reply:
A bigger side effect is that cobra writes the usage to stdout rather than return an error that we can write to our logger. I think that makes the approach unsuitable for us unfortunately.

Reply:
> A bigger side effect is that cobra writes the usage to stdout rather than return an error that we can write to our logger. I think that makes the approach unsuitable for us unfortunately.

This seems like a side consequence that we might want to explore further. I wouldn't use it as an argument not to use cobra's built-in validation. I would say it's stranger to have `< 1` and `> 1` output via two different streams than for both to use the same mechanism and thereby go to the same stream.

Reply:
> I expect that's true, this implementation is consistent with trust-builder so we should update that too.

That's interesting, I'd be curious as to how that came about or if it was simply an oversight. Nonetheless, I agree that they should be consistent.

Reply:
> This seems like a side consequence that we might want to explore further.

[Possible resolution to the inconsistent output streams](https://github.com/buildpacks/pack/pull/701#issuecomment-647617183).

> Nonetheless, I agree that they should be consistent.

It's [fairly](https://github.com/buildpacks/pack/blob/main/internal/commands/create_builder.go#L39) [inconsistent](https://github.com/buildpacks/pack/blob/main/internal/commands/inspect_builder.go#L27-L30) [right](https://github.com/buildpacks/pack/blob/main/internal/commands/inspect_image.go#L30) [now](https://github.com/buildpacks/pack/blob/main/internal/commands/package_buildpack.go#L42), [and](https://github.com/buildpacks/pack/blob/main/internal/commands/rebase.go#L17) [almost](https://github.com/buildpacks/pack/blob/main/internal/commands/trust_builder.go#L19-L22) [exclusively](https://github.com/buildpacks/pack/blob/main/internal/commands/set_default_builder.go#L21-L24) [untested](https://github.com/buildpacks/pack/blob/main/internal/commands/set_run_image_mirrors.go#L19).

Reply:
> A bigger side effect is that cobra writes the usage to stdout rather than return an error that we can write to our logger. I think that makes the approach unsuitable for us unfortunately.

Just to clarify - this means that when a person does the command with invalid args, cobra will print usage to Stdout? Why would you expect it to go to Stderr?

Reply:
> Why would you expect it to go to Stderr?

The command I've encountered wrote to Stdout, however cobra also uses Stderr. I would rather not do a full analysis of Cobra to determine if we'll ever care about Stderr in our use cases. I'd also not like to rely on Cobra's implementation never changing as I could see a justification that this output does belong on Stderr and Cobra could potentially change to that in the future.